ktor-client/ktor-client-tests/common/test/io/ktor/client/tests/MockedTests.kt

```diff
@@ -41,7 +41,8 @@ class MockedTests {
     }
 
     @Test
-    fun testWithLongJson() = testWithEngine(MockEngine) {
+    // TODO: unmute when JS IR serialization is fixed https://github.com/Kotlin/kotlinx.serialization/issues/751
+    fun testWithLongJson() = if (isKotlinJsIr) Unit else testWithEngine(MockEngine) {
```

Comment:
Doesn't it break async test? Does the js test runner look at test result object or at function return type?

Reply:
All tests passed. Static return type is not changed. It is inferred as `dynamic` in both cases.